templates/shared_services/admin-vm/terraform/admin-jumpbox.tf

```diff
@@ -29,7 +29,7 @@ resource "azurerm_windows_virtual_machine" "jumpbox" {
   resource_group_name        = data.azurerm_resource_group.rg.name
   location                   = data.azurerm_resource_group.rg.location
   network_interface_ids      = [azurerm_network_interface.jumpbox_nic.id]
-  size                       = var.admin_jumpbox_vm_sku
+  size                       = "XXX"
```

Comment:
Is that working?

Reply:
It's meant to test the linter and see if it works correctly. I'll revert this when done testing.